lib/shelley/src/Cardano/Wallet/Shelley/Api/Server.hs

```diff
@@ -130,9 +130,10 @@ server
     -> ApiLayer (SeqState n IcarusKey) t IcarusKey
     -> ApiLayer (SeqState n ShelleyKey) t ShelleyKey
     -> IO [PoolId]
+    -> Server (ListStakePools ApiStakePool)
```

Comment:
This is actually more of an `Handler` than a `Server`.

Reply:
I'm considering using `StakePoolLayer` instead, now that the shape of it has settled 🤔

Reply:
Done in cdc4ad7